contextCmds/test/userCmd.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    name: "avatar",
+    type: 2, // 2 is for "USER" | 3 is for "MESSAGE" (EXECUTED BY RIGHT CLICKING EITHER)
```

Comment:
You have to explain this to me too

Reply:
It's a context menu command (right clicking any user-> apps -> command)